cdap-app-fabric/src/main/java/io/cdap/cdap/app/preview/DefaultPreviewRunnerManager.java

```diff
@@ -168,6 +173,17 @@ public void stop(ApplicationId preview) throws Exception {
    */
   @VisibleForTesting
   public Injector createPreviewInjector() {
+    Module previewRunnerRuntimeModule = null;
+    switch(previewRunnerMode) {
```

Comment:
space after switch

Reply:
In general, implementations should not know what "mode" CDAP is running in. Instead, whatever is creating the injector that is used to instantiate this class should be installing the correct module.

Reply:
Resolved offline; decided to remove the refactor for now.